### DISCUSSION TITLE
Is it possible to use a theme from a private repository?

### DISCUSSION THREAD

**Discussion by stefanobartoletti** [slidevjs/slidev#1868] [Q&A]
It is possible to use a shared theme directly from a private GitHub repository, instead of publishing it publicly on NPM?

I want to build a theme that will contain some commercial fonts, that according to their license, I cannot share such a way, but I would still like to be able to have a reusable theming.

I'm thinking about something similar to what is possible with Nuxt layers, in example.

**Comment by kermanx** [comment]
Two possible solutions:

- `npm i https://github.com/your/repo`
- `theme: ./relative/path/` in your `slides.md`'s headmatter.
